brouter-routing-app/src/main/java/btools/routingapp/BRouterView.java

```diff
@@ -635,7 +635,9 @@ private void paintCircle( Canvas canvas, OsmNodeNamed n, int color, int minradiu
     int lat = n.ilat - centerLat;
     int x = imgw / 2 + (int) ( scaleLon * lon );
     int y = imgh / 2 - (int) ( scaleLat * lat );
-    int ir = (int) ( n.radius * 1000000. * scaleLat );
+
+    double[] lonlat2m = CheapRulerSingleton.getLonLatToMeterScales( centerLat );
+    int ir = (int) ( n.radius * scaleLat / lonlat2m[1]);
```

Comment:
I'm really not sure about the scaling here and I don't know much the Android app. I think this is fine, but definitely needs a second look.